src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductCategoryPage.java

```diff
@@ -17,7 +17,9 @@ public ProductTypePage openProductTypePage(ProductCategoryAndSubCategory categor
 
     @Step("ProductCategoryPage: get product category page visibility status by locating page heading")
     public Boolean isOpened() {
-        return $x("//h1[@class = 'portal__heading ng-star-inserted']").is(Condition.visible);
+        return $x("//h1[@class = 'portal__heading ng-star-inserted']")
+                .shouldBe(Condition.exist)
+                .is(Condition.visible);
```

Comment:
the issue with .should() in this case is that it will throw an exception if condition not met, making this method useless
wrap into try{ } catch {return false}